src/facebook.py

```diff
@@ -216,7 +232,9 @@ def request(self, path, args=None, post_args=None):
             else:
                 args["access_token"] = self.access_token
         post_data = None if post_args is None else urllib.urlencode(post_args)
-        file = urllib.urlopen("https://graph.facebook.com/" + path + "?" +
+
+        url = self.url if not self.version else "{}{}/".format(self.url, self.version)
```

Comment:
Consider using named parameters for string formatting. It improves readability, particularly when theres more than one parameter.  (minor detail)